condominium/data/res_config_settings.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="res_config_account_setting_condominium" model="res.config.settings">
+        <field name="group_analytic_accounting" eval="1"/>
+        <field name="group_sale_order_template" eval="1"/>
```

Comment:
don't know what went wrong, but I got this 
![image](https://github.com/odoo/industry/assets/112173839/ffe932ab-25e0-4ec5-a828-07efd84f4854)

Reply:
remove the pricelist from the data